pkg/arvo/lib/push-hook.hoon

```diff
@@ -380,44 +380,36 @@
   ++  push-updates
     |=  =vase
     ^-  (list card:agent:gall)
-    =/  rids=(list resource)  (resource-for-update vase)
-    =|  cards=(list card:agent:gall)
-    |-
-    ?~  rids  cards
-    =/  prefix=path
-      resource+(en-path:resource i.rids)
+    %+  murn  (resource-for-update vase)
```

Comment:
Isn't this missing an `:og`

Reply:
No, because we're calling the arm in the hook core that wraps `resource-for-update:og`.